File: mercadopago-account/balance/dotnet/get_balance.cs
using mercadopago;
using System;
using System.Collections;

public class MyClass {
	public static void Main () {
		MP mp = new MP("CLIENT_ID", "CLIENT_SECRET");

		Hashtable balance = mp.get ("/users/USER_ID/mercadopago_account/balance");

		Console.WriteLine (balance.ToString());
	}
}

File: mercadopago-account/balance/nodejs/get_balance.js
var MP = require ("mercadopago");

var mp = new MP ("CLIENT_ID", "CLIENT_SECRET");

var getBalance = mp.get ("/users/USER_ID/mercadopago_account/balance");

getBalance.then (
    function (balance) {
        console.log (balance);
    },
    function (error) {
        console.log (error);
    });
